Assets/Scenes/MainMenu.unity

```diff
@@ -3406,7 +3406,7 @@ MonoBehaviour:
   m_GameObject: {fileID: 1935625980}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: 0025fdbd4724a23b7bd493093a7e4968, type: 3}
+  m_Script: {fileID: 11500000, guid: b7725820b005fc2afbc3870fef3a04d8, type: 3}
```

Comment:
Why is this changed?

Reply:
Script probably got moved, is for the sound controller.